readme_renderer/__about__.py

```diff
@@ -1,38 +0,0 @@
-# Copyright 2014 Donald Stufft
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-__all__ = [
-    "__title__",
-    "__summary__",
-    "__uri__",
-    "__version__",
-    "__author__",
-    "__email__",
-    "__license__",
-    "__copyright__",
-]
-
-__title__ = "readme_renderer"
-__summary__ = (
-    'readme_renderer is a library for rendering "readme" ' "descriptions for Warehouse"
-)
-__uri__ = "https://github.com/pypa/readme_renderer"
-
-__version__ = "41.0"
-
-__author__ = "The Python Packaging Authority"
-__email__ = "admin@mail.pypi.org"
-
-__license__ = "Apache License, Version 2.0"
-__copyright__ = "Copyright 2014 %s" % __author__
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏do we completely lose this ? I realize it's an invalid metadata key - but I don't think we've recorded the copyright notice anywhere else. Maybe that should at least be reflected in the README.